packages/viz/src/lib/global-interfaces.ts

```diff
@@ -1,18 +1,11 @@
-export enum GeometryType {
-  Point = 'Point',
-  //   MultiPoint = 'MultiPoint',
-  Line = 'Line',
-  //   MultiLine = 'MultiLine',
-  Polygon = 'Polygon'
-  //   MultiPolygon = 'MultiPolygon'
-}
+export type GeometryType = 'Point' | 'Line' | 'Polygon';
 
 export interface NumericFieldStats {
-  name: string;
+  name?: string;
```

Comment:
should not the name at least be mandatory?

Reply:
You catch me :trollface: 

Fixed